src/python/pants/task/testrunner_task_mixin.py

```diff
@@ -408,7 +408,7 @@ def register_options(cls, register):
     # https://github.com/pantsbuild/pants/issues/5073
 
     register('--fast', type=bool, default=True, fingerprint=True,
-             help='Run all tests in a single pytest invocation. If turned off, each test target '
+             help='Run all tests in a single invocation. If turned off, each test target '
                   'will run in its own pytest invocation, which will be slower, but isolates '
```

Comment:
Should `pytest` be removed here as well?

```suggestion
                  'will run in its own invocation, which will be slower, but isolates '
```

Reply:
I didn't even see that! yes.

Reply:
fixed. I had searched using the browser for 'pytest' and didn't even notice the next line!

Reply:
Ah, for some reason the `suggestion` dropped some whitespace leading https://github.com/pantsbuild/pants/pull/7797#discussion_r287132389 - apologies for that.